pages/learn/develop/local-mode.md

```diff
@@ -6,113 +6,119 @@ thumbnail: /img/common/device/running-webterminal-session.png
 
 # Develop locally
 
-Local mode is the development mode for {{ $names.company.lower }}. It allows you to build and sync code to a single development device in your local network without having to go through the whole git push, build, deployment pipeline. It uses the Docker daemon on the device to build container images and starts the container up in very much the same way the {{ $names.company.lower }} device supervisor would.
+Local mode is the development mode for balena. It allows you to build and sync code to a single development device in your local network without having to go through the balenaCloud build service and deployment pipeline. It uses the Docker daemon on the device to build container images and then the supervisor starts the containers in the exact same way as if it were deployed via the cloud.
 
 ## Local mode requirements
 
 In order to use local mode on a device:
-- The device must be running {{ $names.os.lower }} v2.0 or higher with supervisor v4.0 or higher.
-- The device must be running a [development][development] variant of the OS. If you try to use a production variant, you will not be able to use local mode. The production devices have SSH and the Docker socket locked down, both of which are needed for the local mode feature.
-- You must have the [{{ $names.company.lower }} CLI][cli] installed on your workstation.
+- The device must be running balenaOS v2.29.0 or higher.
+- The device must be running a [development][development] variant of the OS.
+- You must have the [balena CLI][cli] installed on your workstation.
 - Local mode must be enabled through the dashboard. To use local mode on a development device, click on the small *Actions* dropdown at the top right of the device page and select *Enable Local Mode*.
 
-__Note:__ At the moment, `{{ $names.company.short }} local push` will not work for multicontainer applications. The recommended development workflow is to put the device in local mode and use the [Docker Compose remote API][compose-remote].
-
 <img src="/img/local-mode/device-in-local-mode.png" width="100%">
 
 ## Scan the network and find your device
 
-Before you can get any code running, you first have to find your device. To do this, login to the {{ $names.company.lower }} CLI and use `{{ $names.company.short }} local scan`.
+Before you can get any code running, you first have to find your device. To do this, login to the balena CLI and use `balena local scan`.
```

Comment:
Might be worth mentioning that `sudo` is often required for this.

Reply:
@CameronDiver yeah that is true. The command signature below actually has it already, but I will quickly add it.

Reply:
fixed in https://github.com/balena-io/docs/pull/1018/commits/948ad4e50c7bd3268d2e2bd0285c36bf87340486